src/main/java/org/checkerframework/specimin/UnsolvedMethod.java

```diff
@@ -38,6 +44,24 @@ public UnsolvedMethod(String name, String returnType, List<String> parameterList
     this.name = name;
     this.returnType = returnType;
     this.parameterList = parameterList;
+    this.isJustMethodSignature = false;
```

Comment:
This constructor should also be chained with the new constructor, rather than having duplicated logic.